Dockerfile

```
@@ -9,7 +9,7 @@ ENV APT_CACHER_NG_VERSION=3.1 \
 
 RUN apt-get update \
  && DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y \
-      apt-cacher-ng=${APT_CACHER_NG_VERSION}* ca-certificates \
+      apt-cacher-ng=${APT_CACHER_NG_VERSION}* ca-certificates curl \
```

I suggest using `wget` here as `curl` has  a huge set of dependencies that bump up the image size quite a bit.

```suggestion
      apt-cacher-ng=${APT_CACHER_NG_VERSION}* ca-certificates wget \
```